pkg/reconciler/eventpolicy/eventpolicy_test.go

```diff
@@ -0,0 +1,155 @@
+/*
+Copyright 2024 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
This is only, because we changed the template some time ago, and it now requires a "tab" instead of 4 spaces, before the `http://www.apache.org/licenses/LICENSE-2.0`